GrandCentralBoard/App/ViewControllers/LoadingViewController.swift

```diff
@@ -6,13 +6,20 @@
 import UIKit
 import GrandCentralBoardCore
 
-let dataDownloader = DataDownloader()
-let availableBuilders: [WidgetBuilding] = [WatchWidgetBuilder(dataDownloader: dataDownloader), BonusWidgetBuilder(dataDownloader: dataDownloader)]
+private let dataDownloader = DataDownloader()
+private let availableBuilders: [WidgetBuilding] = [WatchWidgetBuilder(dataDownloader: dataDownloader), BonusWidgetBuilder(dataDownloader: dataDownloader)]
+private let useLocal = debugBuild
 
 class LoadingViewController: UIViewController {
 
-    let configurationPath = "http://gcb.macoscope.com/configuration.json"
-    let configurationDownloader = ConfigurationDownloader(dataDownloader: DataDownloader())
+    lazy var configurationFetching: ConfigurationFetching = {
+
+        if useLocal {
```

Comment:
No it uses local configuration on `release` build . It should use it on `debug` build.
![screen shot 2016-04-01 at 08 36 07](https://cloud.githubusercontent.com/assets/1786612/14199460/d9f5b660-f7e4-11e5-9bcd-87d8a5ba36b5.png)